.env.template

```
@@ -1,23 +1,27 @@
 # Base URL and App Configurations
 NEXT_PUBLIC_BASE_URL=http://localhost:8001
-NEXT_PUBLIC_SKIP_PRO_CHECK=true
 NEXT_PUBLIC_GITHUB_APP_NAME=dummy
 NEXT_PUBLIC_CONVERSATION_BASE_URL=http://localhost:8001
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 NEXT_PUBLIC_SUBSCRIPTION_BASE_URL=http://localhost:8001
-NEXT_PUBLIC_HMAC_SECRET_KEY=dummy
 NEXT_PUBLIC_WORKFLOWS_URL=http://localhost:8002
 NEXT_PUBLIC_WORKFLOWS_WEBHOOK_URL=https://e7ae-2405-201-d007-30eb-6751-8af2-6edb-b5a4.ngrok-free.app
+NEXT_PUBLIC_SENTRY_REDIRECT_URI=http://localhost:8001/integrations/sentry/callback
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_REDIRECT_URI key should go before the NEXT_PUBLIC_SUBSCRIPTION_BASE_URL key

(UnorderedKey)